src/Perf/IIDOptimizer/Program.cs

```diff
@@ -133,6 +141,12 @@ private static int GuidPatch(string targetAssembly, string outputDirectory, IEnu
                 Console.WriteLine($"\tAssembly : {e.AssemblyReference.Name}"); 
                 return -1; 
             }
+            catch (Exception e)
+            {
+                Console.WriteLine("Application failed with unexpected exception.");

```

Comment:
Consider removing Application for consistency with the other messages.

Reply:
Good point, we should be consistent